### ISSUE TITLE
Examples in colab

### ISSUE THREAD

**Issue by zaccharieramzi** [CEA-COSMIC/pysap-mri#110]
I think we could benefit from having the examples written as colab notebooks:
- potential use of GPU for GPU related examples
- showing the expected results of the examples (nice figures)
- easy to use pysap-mri without having to install it, to ease user adoption

It can be done one by one and I think for people getting in pysap-mri it can be a good starting point.

**Comment by chaithyagr** [comment]
Assigning this to myself, will get at this in Auguest first week. I will also update pysap-tutorials for the same

**Comment by philouc** [comment]
@zaccharieramzi you're absolutely right and we have discussed that today with @chaithyagr. We plan to add specific example for density compensation in `pysap-tutorials` based on the GPUnufft. The pysap-tutorials repository is already in place to be compatible with binder and Colab. I can also contribute to that in early August. Then, we'll introduce a specific gallery with nice images as done for instance in nilearn. This will attract more potential users.
